app/src/main/res/layout/item_task.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+
+    </data>
+
+    <androidx.cardview.widget.CardView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@drawable/background_solitude_10dp_corners"
+            android:paddingVertical="@dimen/margin_xmedium">
+
+            <androidx.appcompat.widget.AppCompatCheckBox
+                android:id="@+id/checkboxTaskItem"
+                android:layout_width="0dp"
+                android:layout_height="match_parent"
+                android:layout_marginStart="@dimen/margin_small"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:checked="true" />
+
+            <androidx.appcompat.widget.AppCompatTextView
+                android:id="@+id/textTaskItem"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/margin_medium"
+                android:gravity="start"
+                android:textAppearance="@style/Text.H5.Secondary.Roboto.Regular"
+                android:textColor="@color/eclipse"
+                android:textSize="@dimen/text_h2"
```

Comment:
Możesz definiować wielkość i kolor tekstu od razu w `textAppearance` tak jak to zrobiłeś u góry. Tam wszystko jest zdefiniowane: 
 H5 to wielkość,
 Secondary to kolor
Roboto to czcionka
Regular to styl pisania,
Na razie nie mam wszystkich możliwych kombinacji zdefiniowanych, jeśli takiego jaki potrzebujesz nie ma to zrób wzorujac sie na innych ;)